gateway/utils/metrics/projects.go

```diff
@@ -9,103 +9,94 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/spaceuptech/space-api-go/types"
 
+	"github.com/spaceuptech/space-cloud/gateway/config"
 	"github.com/spaceuptech/space-cloud/gateway/utils"
 )
 
-func (m *Module) generateMetricsRequest() (string, map[string]interface{}, map[string]interface{}, bool) {
+func (m *Module) generateMetricsRequest(project *config.Project, ssl *config.SSL) (string, string, map[string]interface{}, map[string]interface{}) {
```

Comment:
Good to name all parameters
```suggestion
func (m *Module) generateMetricsRequest(project *config.Project, ssl *config.SSL) (clusterID, projectID string, set map[string]interface{}, min map[string]interface{}) {
```